### DISCUSSION TITLE
School blocked extension settings

### DISCUSSION THREAD

**Discussion by spderman3333** [3kh0/ext-remover#894] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
The title says it all, LTMEAT FLood Freeze still work because step 5 needs me to flip the "Allow access to file URLs" switch? (I'm too afraid to try it again as I got it to work right before they blocked it.)
Thanks in advance.

**Comment by S-PScripts** [comment]
use a different exploit

**Reply by schoolexploitkid** [reply]
pls dont necropost

**Reply by S-PScripts** [reply]
i wanted to mark an answer instead of just closing

**Comment by AshtonDavies** [comment, marked as answer]
Shrey has shared a method that works without extension settings:


SETUP: 
1. Go to chrome-extension://ext-id-here/manifest.json
2. Bookmark it 
3. Go to chrome://bookmarks 
4. Make a folder & dupe the shit out of the thing in the folder 


EXECUTION 
1. Sign in to your chromebook with your internet OFF 
2. Open the folder that you created in a new window 
3. When the tabs start responding, close them ALL
4. Open the folder in a new window again 
5. The pages should hang. Close them all, then turn wifi back on
